src/batch-league-flysystem/src/Job/MoveFilesJob.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yokai\Batch\Bridge\League\Flysystem\Job;
+
+use Closure;
+use League\Flysystem\FilesystemException;
+use League\Flysystem\FilesystemOperator;
+use League\Flysystem\FilesystemWriter;
+use League\Flysystem\UnableToDeleteFile;
+use League\Flysystem\UnableToReadFile;
+use League\Flysystem\UnableToWriteFile;
+use Yokai\Batch\Exception\UnexpectedValueException;
+use Yokai\Batch\Job\JobInterface;
+use Yokai\Batch\Job\Parameters\JobParameterAccessorInterface;
+use Yokai\Batch\JobExecution;
+
+/**
+ * This job allows you to move files from one filesystem ot another.
+ */
+class MoveFilesJob implements JobInterface
```

Comment:
This class look like CopyFilesJob#execute instead of FilesystemOperator attribute and file deletion l.55.

Perhaps we could use the implementation of a visitor to limit code duplication?

Reply:
I was thinking about that, but I wasn't able to find a way to factorise the code between those job.
Any advice on the structure?

Reply:
No, sorry but I couldn't find one.
I searched when I did the review but couldn't find it.
So, I just submitted the idea, if ever you had more than me.